### PR TITLE
Feat : added main pages

### PR DESCRIPTION
**Pull request by nishasen**
It contains
1. note
2. archive
3.trash page

### CODE REVIEW

File: wiznotes/src/Pages/MainPages/Trash.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './MainPages.css';
+import { Wallpaper, Topnav, SideNavigation, TrashPage } from '../../Components';
+
+const Trash = () => {
+  //const {page, setPage} = usePage();
```
Comment (sadathshariff) [inline wiznotes/src/Pages/MainPages/Trash.jsx]: remove commented code here

**Review by sadathshariff — APPROVED** [review]
Overall looks good , go ahead and merge

**Review by srejitk — APPROVED** [review]
No issues found. Like how you've made the hamburger with hide functionality. 
Clean out the comments though.

File: wiznotes/src/Pages/MainPages/Trash.jsx (hunk 1 above)
Comment (nishasen) [reply wiznotes/src/Pages/MainPages/Trash.jsx]: done

**Pull request merged**